src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -346,4 +351,11 @@ codeunit 149005 "BCPT Line"
         Parm := format(FldRef.Value, 0, 9);
         exit(true);
     end;
+
+    local procedure OnBeforeAddLogEntry(var BCPTLine: Record "BCPT Line"; var Operation: Text; var ExecutionSuccess: Boolean; var Message: Text; var Handled: Boolean)
```

Comment:
it's not an event so the name is misleading. If its sole purpose is to raise the event, then I would call it RaiseOnBlaBlaBla.
but I still wonder why you're not raising the event directly...

Reply:
Yeah, this is misleading but I have a reason for that.
The ENTIRE object has access = internal. 
Why sole idea was to raise this event directly! But since the object has access = internal you can't subscribe to the event from another app which is the purpose of the entire change.

Hence why I did this wrapper thingy workaround. Another solution would have been to do the event directly, subscribe to it in "BCPT Tes Suite" and then call another event which can be reached from other apps since "BCPT Test Suite" does not have internal accesss. But I thought that option would be confusing for people who looks for events and reads the code. There would be no direct calling thread to the event in "BCPT Test  Suite" which takes away the readability of the change.